client/context/context.go

```diff
@@ -163,6 +164,12 @@ func GetAccountDecoder(cdc *codec.Codec) auth.AccountDecoder {
 	}
 }
 
+// WithKeybase returns a copy of the context with an updated keybase.
+func (ctx CLIContext) WithKeybase(kb cryptokeys.Keybase) CLIContext {
```

Comment:
Does this PR need to be rebased somehow with #3514? I recall we dont have a Keybase as part of a CLIContext.

Reply:
Let me clean this up. There has been too much parallel work.